ch08.adoc

```diff
@@ -675,3 +675,130 @@ The data creator shall specify the floating-point arithmetic precision used duri
 Using the given computational precision in the interpolation computations is a necessary, but not sufficient, condition for the data user to be able to reconstitute the coordinates to an accuracy comparable to that intended by the data creator.
 For instance, a **`computational_precision**` value of **`"64"**` would specify that, using the same implementation and hardware as the creator of the compressed dataset, sufficient accuracy could not be reached when using a floating-point precision lower than 64-bit floating-point arithmetic in the interpolation computations required to reconstitute the coordinates.
 
+[[lossy-compression-via-quantization, Section 8.4, "Lossy Compression via Quantization"]]
+=== Lossy Compression via Quantization
+
+Geoscientific models and measurements generate false floating-point precision (scientifically meaningless data bits) that wastes storage space.
+False precision can mislead (by implying noise is signal) and is scientifically pointless.
+Quantization algorithms can eliminate false precision, usually by rounding the least significant bits of [<<IEEE_754>>] floating-point mantissas to zeros.
+(Quantization of integer types, although theoretically allowed, is not covered by this convention.)
+The quantized results are valid [<<IEEE_754>>] values---no special software or decoder is necessary to read them.
+Importantly, the quantized bits compress more efficiently than random bits.
+Thus quantization is sometimes referred to as a form of lossy compression although, strictly speaking, quantization only pre-conditions data for more efficient compression by a subsequent compressor.
+
+The CF conventions of this section define a metadata framework to record quantization properties alongside quantized floating-point data variables.
+The goals are twofold.
+First, to inform interested users how, and to what degree, the quantized data differ from the original unquantized data, which are not stored in the dataset and may no longer exist.
+Second, to provide the necessary provenance metadata for users to reproduce the data transformations on the same or other raw data.
+These conventions also allow users to better understand the precision that data producers expect from source models or measurements.
+Use of these conventions ensures that all quantized variables are clearly marked as such, and thus alerts users to cases where these guidelines have not been followed.
```

Comment:
I intended the second phrase of the last sentence, "Use of these conventions ensures that all quantized variables are clearly marked as such, **and thus alerts users to cases where these guidelines have not been followed**." to convey that since all quantized variables must be marked as such, it will be easier for users to identify cases where the dataset producers inadvertently quantized variable that should not be quantized, e.g., non-data variables, coordinate variables or formula terms variables. In practice, it is easy to quantize every floating point variable in a dataset, and what is hard is to refrain from quantizing variables that should not be quantized. (And to refrain from quantizing _FillValue, for that matter). However, since all quantized variables will be annotated as such, then discerning users should be able to recognize that unexpected problems using the data might arise from prohibited quantization of protected variables. That is the intent.
Our options include removing the second phrase (in bold), leaving it as is, or changing it to clarify the intent. What do you suggest?

Reply:
Thanks - I see where you're going now! However, I still don't think we need this sentence :). This is because this it is an error for which the resolution is unclear. Do you (Does your software library) ignore quantization information on a coordinate variable and carry on using the dataset assuming that the coordinates are un-quantized?, or do you stop using the dataset because the coordinates are wrong? There's no right answer! In any event, the canonical way to tell if the file non-conformant is a cf-checker, which will always raise an error for this case.

Reply:
If a dataset attempts to be CF compliant but fails because the creator inadvertently quantized a coordinate variable, the CF metadata might alert the user to the issue (that the coordinate is quantized). Most geoscientists never have nor ever will use a CF checker. The convention gives them a better chance to see potential issues. Perhaps you're right, it's pretty "meta" so I'll take out the last sentence.